pkg/curatedpackages/packageclient.go

```diff
@@ -116,7 +117,56 @@ func (pc *PackageClient) InstallPackage(ctx context.Context, bp *packagesv1.Bund
 	if err != nil {
 		return err
 	}
-	fmt.Println(&stdOut)
+	fmt.Print(&stdOut)
+	return nil
+}
+
+func (pc *PackageClient) ApplyPackages(ctx context.Context, fileName string, kubeConfig string) error {
+	params := []string{"apply", "-f", fileName, "--kubeconfig", kubeConfig}
+	stdOut, err := pc.kubectl.ExecuteCommand(ctx, params...)
+	if err != nil {
+		fmt.Print(&stdOut)
+		return err
+	}
+	fmt.Print(&stdOut)
+	return nil
+}
+
+func (pc *PackageClient) CreatePackages(ctx context.Context, fileName string, kubeConfig string) error {
+	params := []string{"create", "-f", fileName, "--kubeconfig", kubeConfig}
+	stdOut, err := pc.kubectl.ExecuteCommand(ctx, params...)
+	if err != nil {
+		fmt.Print(&stdOut)
+		return err
+	}
+	fmt.Print(&stdOut)
+	return nil
+}
+
+func (pc *PackageClient) DeletePackages(ctx context.Context, args []string, kubeConfig string) error {
```

Comment:
can we make this method signature a bit more explicit? I would suggest splitting `args` into multiple method arguments with explicit names.
Right now, this is creating a coupling from the cmd command to the kubectl command passing through this method

Reply:
args is not a method argument but a customer input. I could name it in a better way but it is supposed to represent the packages the customer needs. I'll update the name.

Reply:
if they are package names wanted by the user, can it be just be called `packageNames`?